tests/test_tutorial.py

```diff
@@ -243,7 +240,7 @@ def test_tutorial(self):
         'repository', 'targets', 'myproject', 'file4.txt'))
     octal_file_permissions = oct(os.stat(target4_filepath).st_mode)[4:]
     custom_file_permissions = {'file_permissions': octal_file_permissions}
-    repository.targets.add_target(target4_filepath, custom_file_permissions)
+    repository.targets.add_target('myproject/file4.txt', custom_file_permissions)
```

Comment:
Same as (several comments) above, does this pattern look a little strange?

Reply:
Same as comment above: https://github.com/theupdateframework/tuf/pull/1008/commits/6d8a84d9938d149e01c3bd0bfb52d056bb5b7b55